### PR TITLE
added images to product page

### PR DESCRIPTION
**Pull request by BoyanSiromahov**

### CODE REVIEW

File: src/main/java/model/Product.java [hunk 1]
```diff
@@ -20,8 +28,9 @@
 
     public Product() { }
 
-    public Product(String name, String description, int inventory, byte[] image, Long shopID)
+    public Product(String name, String description, int inventory, byte[] image, Long shopID, String imageUrl)
```
Comment (simonkrol) [inline src/main/java/model/Product.java]: If you're using the imageUrl String, do we still need the image byte array?

File: src/test/java/model/ProductIntegrationTest.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ public void setUp()
     {
         List<Product> prods = new ArrayList<Product>();
         shop = new Shop("Amazon", prods);
-        product = new Product("Name","Description", 20, "testPhoto".getBytes(), shop.getId());
+        product = new Product("Name","Description", 20, "testPhoto".getBytes(), shop.getId(), "https://dpa730eaqha29.cloudfront.net/mltnews/wp-content/uploads/2011/03/Events-in-Seattle-Starbucks-Coffee-Company.jpg");
```
Comment (simonkrol) [inline src/test/java/model/ProductIntegrationTest.java]: Lol does this mean that the images we use are just pulled/hosted somewhere else?

**Review by simonkrol — APPROVED** [review]

File: src/main/java/model/Product.java (hunk 1 above)
Comment (BoyanSiromahov) [reply src/main/java/model/Product.java]: no we dont, forgot to remove it.

File: src/test/java/model/ProductIntegrationTest.java (hunk 2 above)
Comment (BoyanSiromahov) [reply src/test/java/model/ProductIntegrationTest.java]: yep. Easiest way to transfer images over.

**Pull request merged**